{"uid":"a22bfcf6f74b01b6","name":"test_address_prefix[Test for Turing]","fullName":"tests.test_network_parameters.TestNetworkPrefix#test_address_prefix","historyId":"62fb4c56335a1fd3a56ee61c95b01665","time":{"start":1669406885757,"stop":1669406886434,"duration":677},"status":"broken","statusMessage":"TimeoutError: [Errno Can't connect to all nodes of network] Turing","statusTrace":"self = <tests.test_network_parameters.TestNetworkPrefix object at 0x7f0f0230b070>\nchain = <tests.utils.chain_model.Chain object at 0x7f0f0235b370>\n\n    def test_address_prefix(self, chain: Chain):\n>       chain.create_connection()\n\ntests/test_network_parameters.py:15: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <tests.utils.chain_model.Chain object at 0x7f0f0235b370>\n\n    def create_connection(self):\n        for node in self.nodes:\n            if (self.substrate):\n                return self.substrate\n            try:\n                self.substrate = create_connection_by_url(node.get('url'))\n                return self.substrate\n            except:\n                print(\"Can't connect to that node\")\n                continue\n    \n>       raise TimeoutError(\"Can't connect to all nodes of network\", self.name)\nE       TimeoutError: [Errno Can't connect to all nodes of network] Turing\n\ntests/utils/chain_model.py:29: TimeoutError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"broken","statusMessage":"TimeoutError: [Errno Can't connect to all nodes of network] Turing","statusTrace":"self = <tests.test_network_parameters.TestNetworkPrefix object at 0x7f0f0230b070>\nchain = <tests.utils.chain_model.Chain object at 0x7f0f0235b370>\n\n    def test_address_prefix(self, chain: Chain):\n>       chain.create_connection()\n\ntests/test_network_parameters.py:15: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <tests.utils.chain_model.Chain object at 0x7f0f0235b370>\n\n    def create_connection(self):\n        for node in self.nodes:\n            if (self.substrate):\n                return self.substrate\n            try:\n                self.substrate = create_connection_by_url(node.get('url'))\n                return self.substrate\n            except:\n                print(\"Can't connect to that node\")\n                continue\n    \n>       raise TimeoutError(\"Can't connect to all nodes of network\", self.name)\nE       TimeoutError: [Errno Can't connect to all nodes of network] Turing\n\ntests/utils/chain_model.py:29: TimeoutError","steps":[],"attachments":[{"uid":"b2afade988bcd3fd","name":"stdout","source":"b2afade988bcd3fd.txt","type":"text/plain","size":54}],"parameters":[],"attachmentsCount":1,"shouldDisplayMessage":true,"hasContent":true,"stepsCount":0},"afterStages":[],"labels":[{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_network_parameters"},{"name":"subSuite","value":"TestNetworkPrefix"},{"name":"host","value":"fv-az424-565"},{"name":"thread","value":"1738-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_network_parameters"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"chain","value":"<tests.utils.chain_model.Chain object at 0x7f0f0235b370>"}],"links":[],"hidden":true,"retry":true,"extra":{"categories":[],"tags":[]},"source":"a22bfcf6f74b01b6.json","parameterValues":["<tests.utils.chain_model.Chain object at 0x7f0f0235b370>"]}